src/snakemake/report/__init__.py

```
@@ -541,21 +547,34 @@ def get_time(rectime, metatime, sel_func):
+                        name=await expand_cat_name(report_obj.category),
+                        wildcards=wildcards,
+                        job=job,
                     )
                     subcategory = Category(
-                        name=report_obj.subcategory, wildcards=wildcards, job=job
+                        name=await expand_cat_name(report_obj.subcategory),
+                        wildcards=wildcards,
+                        job=job,
                     )
-                    labels = expand_labels(report_obj.labels, wildcards, job)
+                    labels = await expand_labels(report_obj.labels, wildcards, job)
```

Ruff (0.8.2)

Function definition does not bind loop variable `report_obj`

(B023)